Analysis/src/BaseEventAnalyzer.cpp

```diff
@@ -61,12 +61,15 @@ EventCategorySet BaseEventAnalyzer::DetermineEventCategories(EventInfoBase& even
                                                                           {DiscriminatorWP::Medium, 0},
                                                                           {DiscriminatorWP::Tight, 0}};
     EventCategorySet categories;
-
-    const bool is_boosted = event.SelectFatJet(cuts::hh_bbtautau_2016::fatJetID::mass,
-                                               cuts::hh_bbtautau_2016::fatJetID::deltaR_subjet) != nullptr;
```

Comment:
keep the definition of boosted as it was. Implement workaround inside SelectFatJet for 2018